docs/components/select.md

```diff
@@ -252,6 +252,10 @@ of the input, dropdown, and selected item.
 />
 ```
 
+### positionFixed
```

Comment:
Again, unsure if props should be `camelCase` or `PascalCase` ? When I'm looking through the docs, it appears we aren't consistent between `.md` pages 🙄

Reply:
Camel Case ftw!